src/Component/QueryEditor.tsx

```diff
@@ -102,7 +102,7 @@ export const QueryEditor: ComponentType<Props> = ({ datasource, onChange, onRunQ
                 language="json"
                 showLineNumbers={true}
                 showMiniMap={payload.length > 100}
-                value={payload}
+                value={isString(payload) ? payload : JSON.stringify(payload,undefined,2)}
```

Comment:
is there any advantage of isString of this?
```suggestion
                value={typeof payload === 'string' ? payload : JSON.stringify(payload,undefined,2)}
```

if not, I'd like to keep coupling with lodash as small as possible.

Reply:
I don't know who is better. It's OK to use typeof directly.

Reply:
I believe the `typeof` check is sufficient. There's `new String(...)` construct in jabbascript but we don't use that. 
https://github.com/lodash/lodash/blob/2f79053d7bc7c9c9561a30dda202b3dcd2b72b90/isString.js